lib/dai-plugin-mcd/test/ManagedCdp.spec.js

```diff
@@ -90,20 +91,20 @@ async function expectUtilValues(cdp, { val, ratio, isSafe, dai }) {
     cdp.isSafe(),
     cdp.getMinSafeCollateralAmount(),
     cdp.getCollateralAvailable(),
-    cdp.getDaiAvailable()
+    cdp.getDaiAvailable(),
+    cdp.type.getLiquidationRatio()
   ]);
-
-  const minVal = debtValue.div(val);
+  const minVal = debtValue.times(liquidationRatio).div(val);
 
   if (val !== undefined) {
-    expect(collateralValue.eq(val)).toBeTruthy();
+    expect(collateralValue.toNumber()).toBe(val);
     expect(minCollateral.toNumber()).toBe(minVal.toNumber());
```

Comment:
I did this because, for some reason, the .eq wasn't always returning true (even though it seems like it should have - could it be a bug in .eq?)